api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -83,4 +90,7 @@ message DnsCacheConfig {
   // this is used as the cache's DNS refresh rate when DNS requests are failing. If this setting is
   // not specified, the failure refresh rate defaults to the dns_refresh_rate.
   config.cluster.v3.Cluster.RefreshRate dns_failure_refresh_rate = 6;
+
+  // The config of circuit breakers for resolver. It provides a configurable threshold.
```

Comment:
What is the default if this is not specified? How are we going to roll this out with the existing circuit breakers?

Reply:
I'm planning to use this if users configured `dns_cache_circuit_breakers`. if not, we use the default circuit breakers derived from cluster config.

Reply:
Update the docs to spell that out please.

Reply:
Also, if runtime works for overrides that also needs to be documented.

Reply:
I'm still not sure what the default here is if not set? Please make the docs more clear.